src/libraries/Microsoft.Extensions.Logging.Console/src/SimpleConsoleFormatter.cs

```diff
@@ -157,8 +158,12 @@ private static string GetLogLevelString(LogLevel logLevel)
 
         private ConsoleColors GetLogLevelConsoleColors(LogLevel logLevel)
```

Comment:
Would it be better to have a mobile specific .cs file that implemented a function that returned true for `disableColors`?

Reply:
We don't build RID specific versions of these libraries, and don't really plan on adding them. It impacts the build times of these libraries.

See also https://github.com/dotnet/runtime/issues/53900.

cc @ViktorHofer

Reply:
Yes, we don't want OOB libraries to have runtime specific build configurations. Instead they are better of using runtime configurations when possible.